adapters/appnexus/appnexus.go

```diff
@@ -302,10 +303,11 @@ func (a *AppNexusAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ad
 		}
 	}
 	for i := 0; i < len(request.Imp); i++ {
-		memberId, err := preprocess(&request.Imp[i], defaultDisplayManagerVer)
+		memberId, impAdPodId, err := preprocess(&request.Imp[i], defaultDisplayManagerVer)
 		if memberId != "" {
 			memberIds[memberId] = true
 		}
+		adPodId = adPodId || impAdPodId
```

Comment:
Should this produce an error or assume it's enabled for all imps? Does this adapter have an existing pattern to follow if there's missing or conflicting data in an imp?

Reply:
There is no consistent pattern, it now throws bidder level error.

Reply:
I think the ideal situation is, if this flag is on in at least one Imp, and either missing or on in the rest, then we generate Ids. If some have it on and others have it off, then we throw an error. If it is just off or missing, then no Ids. This really should be a request level flag rather than an Imp level flag, but we don't have request level bidder parameters.

Actually, now that I think of it, there was a proposal for request level bidder params, that would be merged into all the imp level bidder params. I don't recall that it was ever implemented.

Reply:
`if this flag is on in at least one Imp, and either missing or on in the rest, then we generate Ids. If some have it on and others have it off, then we throw an error. If it is just off or missing, then no Ids.` - yes, this is is how it works now.

I don't think we have this functionality. `request level bidder params` - sounds like a new feature.

Reply:
Yes, I don't think it was ever implemented. But as I noted in my approval, you need all Imps set to true to generate ad pod IDs. If at least one imp is set to generate ad pod IDs, and another is left blank, this will error out.